docs/examples/vector_stores/JaguarIndexDemo.ipynb

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
<p>could you break down this into different sections and add outputs for each step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9754/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>